bin/harvey-transformer

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env node
+
+var fs = require('fs')
+var pkg = require('../package.json');
+var commandLine = require('commander');
+var options = getCommandLineArguments();
+var loadJson = require('../lib/util/loadJson.js');
+
+var reporterFactory = require('../lib/reporters/reporterFactory.js');
+var reporter = (options.reporter === 'none') ? null : reporterFactory.createReporter(options.reporter || 'console');
+
+if (!reporter) {
+	fail('No reporter specified')
+}
+
+var config;
+try {
+	config = (options.configFile) ? loadJson(options.configFile) : {};
+} catch(ex) {
+	fail(ex);
+}
+
+if (!options.jsonResults) {
+	commandLine.help()
+}
+
+var overallResults = loadJson(options.jsonResults)
+overallResults.timeStarted = new Date(overallResults.timeStarted)
+overallResults.timeEnded = new Date(overallResults.timeEnded)
+
+overallResults.suiteResults = overallResults.suiteResults.map(function(result){
+	result.timeStarted = new Date(result.timeStarted)
+	result.timeEnded = new Date(result.timeEnded)
+	return result
+})
+
+reporter.reportResults(overallResults, config, process.stdout, function(error) {
```

Comment:
@tschwecke what is the thought for passing `config` along to the `reportResults` method? None of the built-in reporters use it now. If we eliminate it, `harvey-transformer` won't have to load a config file.